src/patient-registration/input/input.css

```diff
@@ -82,3 +82,35 @@
 .identifierInput {
     height: 80px !important;
 }
+
+.searchInput {
+  width: 660px !important;
+  height: 40px !important;
+}
+
+.searchResults {
+  height: auto;
+  margin-top: .25rem;
+  border: 1px solid #ababab;
+  border-radius: .25rem;
+  background-color: white;
+  position: absolute;
+  width: 660px;
```

Comment:
Maybe cap it with max-width: 100% or so - otherwise we may have issues on mobile.